view/template/page/events.php

```diff
@@ -0,0 +1,31 @@
+<?php Response::setMetaDescription(__('LBRY Welcomes You')) ?>
+<?php Response::setMetaTitle(__('LBRY Welcomes You')) ?>
+<?php echo View::render('nav/_header', ['isDark' => true, 'isAbsolute' => true]) ?>
+<main>
+  <div class="cover cover-dark cover-center cover-full cover--dark-overlay" style="background-image: url(/img/table2.jpg)">
+    <div class="content content-dark text-center">
+      <h1 class="cover-title cover-title-tile cover-item--outline">
+        LBRY Welcomes You To The Blockchain Event
+      </h1>
+      <h3 class="cover-item--outline">We're excited to share about LBRY with you!
+      </h3>
+      <div class="spacer1">
+        <a href="#about" class="btn-primary btn-large">Tell Me More!</a>
+      </div>
+    </div>
+  </div>
+  <div class="cover cover-light" id="about">
+    <div class="content content-light content-readable">
+      <h3>Enter your email address to receive LBRY updates, news, and information for India's LBRYians!</h3>
+      <p>As an open source project, our development and future relies on our communities around the world, and we're delighted to see our community growing in India. Please enter your email below and we'll give you instructions to redeem the LBC as mentioned by our team.</p>
+      <?php echo View::render('mail/_subscribeForm', [
+        'tag' => 'events',
```

Comment:
Can we change this tag to something else? @ceoger  / @jsigwart . Maybe something more specific.

Reply:
not sure, this is not a specific event, there is an exchange launch happening in new delhi for which we plan to pass by, we may get to talk to the attendees about LBRY, so maybe we can use an existing landing page else please suggest a any?

Reply:
What's the purpose of tagging these people? Will you reach out to them? Maybe make a generic one like `india-other` for these one offs? Just didn't want a tag in our system called "events" - too generic.